freebridge-ai/main.py

```
@@ -0,0 +1,78 @@
+system_prompt = (
+    "당신은 법률/계약 전문 AI 보조원입니다. 제공된 문맥(Context)을 바탕으로 답변하세요. "
+    "모르는 내용은 지어내지 마세요.\n\n{context}"
+)
+
+prompt = ChatPromptTemplate.from_messages([
+    ("system", system_prompt),
+    ("human", "{input}"),
+])
+
+# 문서 결합 및 리트리벌 체인 생성
+combine_docs_chain = create_stuff_documents
```

Ruff (0.15.2)

[error] Undefined name `create_stuff_documents`

(F821)